code/game/machinery/buttons.dm

```diff
@@ -56,6 +56,12 @@
 		else
 			icon_state = skin
 
+/obj/machinery/button/allowed(mob/M)
+	if(req_access == 150 && issilicon(M)) //The syndicate figured out how to lock out borgs

```

Comment:
isn't req_access a list? also use ACCESS_SYNDICATE over a magic number, you can use defines outside of maps